src/Exceptions/HttpExpectationFailed.php

```diff
@@ -8,4 +8,3 @@ class HttpExpectationFailed extends HttpException {
     public const DESCRIPTION = "Indicates that the expectation given in the request's Expect header field could not be met by at least one of the inbound servers.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/417";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces